test/integration/scheduler/suite_test.go

```diff
@@ -71,8 +73,15 @@ func managerAndSchedulerSetup(ctx context.Context, mgr manager.Manager) {
 	err := indexer.Setup(ctx, mgr.GetFieldIndexer())
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
+	transformations := []config.ResourceTransformation{
+		{
+			Input:    corev1.ResourceName(pseudoCPU),
+			Strategy: config.Replace,
+			Outputs:  corev1.ResourceList{corev1.ResourceCPU: resourcev1.MustParse("2")},
+		},
+	}
 	cCache := cache.New(mgr.GetClient())
-	queues := queue.NewManager(mgr.GetClient(), cCache)
+	queues := queue.NewManager(mgr.GetClient(), cCache, queue.WithResourceTransformations(transformations))
```

Comment:
Instead of passing the transformations by default, could we set the transformations when we enable this feature since this feature is still in the alpha stage?

I guess that we can implement the transformations dedicated `ginkgo.Describe`.

Reply:
>  I guess that we can implement the transformations dedicated ginkgo.Describe

Sorry,  I don't understand what you are suggesting about using ginkgo.Describe.  I don't see a clean way to only set the manager's workloadInfoOptions field for a single test case where the feature is enabled, but I am definitely not aware of all the tricks one could try to use with ginkgo.